models/article-model.js

```diff
@@ -16,26 +16,33 @@ exports.fetchArticleByID = (articleID) => {
     });
 };
 
-exports.fetchAllArticles = (sort_by = "created_at", order = "DESC") => {
+exports.fetchAllArticles = (topic, sort_by = "created_at", order = "DESC") => {
   const validSortBys = ["title", "topic", "author", "votes", "created_at"];
   const validOrderBys = ["ASC", "DESC"];
+  const validTopics = ["mitch", "cats", "coding", "football", "cooking"];
```

Comment:
If you were looking to be able to add topics this would need to be dynamic, perhaps a function to check if a topic exists, for now though this will work